harvester/harvester
# Remove unused file nfs.go

**Problem:**
`pkg/controller/master/backup/nfs.go` is not referenced by any file. We've already changed to use LH [backupstore](https://github.com/longhorn/backupstore) to operate NFS/S3 backup.

**Solution:**
Remove it and check whether NFS backup can work.

**Related Issue:**
https://github.com/harvester/harvester/issues/1908

**Test plan:**
1. Setup NFS backup target.
2. Create a VM and create a file with content `test-123`.
3. Run `sync` command.
4. Create a VMBackup. It should be created without any error.
5. Modify file content in the VM.
6. Restore the VMBackup to the existing/new VM.
7. The file in the existing/new VM should have content `test-123`.
